packages/library-base/src/jvm/kotlin/io/realm/kotlin/internal/platform/SystemUtils.kt

```diff
@@ -46,6 +51,62 @@ public actual fun prepareRealmFilePath(directoryPath: String, filename: String):
     return File(directoryPath, filename).absolutePath
 }
 
+private fun copyStream(inputStream: InputStream, outputStream: OutputStream) {
+    @Suppress("MagicNumber")
+    val buf = ByteArray(4096)
```

Comment:
We could make the 4096 a constant FOUR_KB or something similar.

Reply:
It is only used for this copy and doesn't have connections to other parts, and does not even appear anywhere else in our codebase, so I would deem it ok to just suppress it.

Reply:
Maybe rename to `FILE_COPY_BUFFER_SIZE` or something like that, which indicates the behaviour rather than the value